src/www/containers/contactpage/ContactPage.jsx

```diff
@@ -0,0 +1,140 @@
+import "./ContactPage.scss";
+
+import React, { Component } from "react";
+
+import Input from "../../components/input/Input";
+import axios from "axios";
+import { withAlert } from "react-alert";
+import { withTranslation } from "react-i18next";
+
+const form_img = "/img/form_img.png"; //Photo by LinkedIn Sales Navigator on Unsplash
+
+export default withAlert()(
+  withTranslation()(
+    class ContactPage extends Component {
+      state = {
+        firstName: "",
+        lastName: "",
+        email: "",
+        message: "",
+        serverMessage: "",
+        check_success: false,
+        firstΝameHighlight: false,
+        lastNameHighlight: false,
+        emailHighlight: false
+      };
+
+      clearHighlights = async () => {
+        this.setState({
+          firstNameHighlight: false,
+          lastNameHighlight: false,
+          emailHighlight: false
+        });
+      };
+
+      handleResponse = (data, target) => {
+        const { t, alert } = this.props;
+        if (data.error) {
+          alert.error(data.error);
+        } else {
+          alert.success(t("contactpage.thanks"));
+          target.reset();
+        }
+      };
+
+      contact = (firstName, lastName, email, message, target) => {
+        axios
+          .post("/contact/submit", {
+            firstName,
+            lastName,
+            email,
+            message
+          })
+          .then(res => this.handleResponse(res.data, target))
+          .catch(err => this.handleResponse(err.response.data, target));
+      };
+
+      handleSubmit = event => {
+        event.preventDefault();
+
+        const { firstName, lastName, email, message } = this.state;
+
+        if (
+          firstName !== "" &&
+          lastName !== "" &&
+          email !== "" &&
+          message !== ""
+        ) {
+          this.clearHighlights();
+          this.contact(firstName, lastName, email, message, event.target);
+        } else {
+          this.setState({
+            firstΝameHighlight: firstName === "",
+            lastNameHighlight: lastName === "",
+            emailHighlight: email === ""
+          });
+        }
+      };
+
+      render() {
+        const { t } = this.props;
+        return (
+          <div className="container">
+            <span className="contact-form-title1">
```

Comment:
This should be some sort of header (h2/h3), and not span. (similarly with the element below)